core/domain/suggestion_services_test.py

```diff
@@ -1648,3 +1648,548 @@ def test_get_voiceover_application_from_model_with_invalid_type_raise_error(
+
+    def test_create_translation_suggestions_eq_lang_increases_translation_count(
+            self):
+        self._assert_community_contribution_stats_is_in_default_state()
+
+        self._create_translation_suggestion_with_language_code('hi')
+        self._create_translation_suggestion_with_language_code('hi')
+
+        stats = suggestion_services.get_community_contribution_stats()
+        self.assertEqual(stats.question_reviewer_count, 0)
+        self.assertEqual(stats.question_suggestion_count, 0)
+        self.assertDictEqual(
+            stats.translation_suggestion_counts_by_lang_code, {'hi': 2})
+        self.assertDictEqual(
+            stats.translation_reviewer_counts_by_lang_code, {})
+
+    def test_accept_translation_suggestion_lowers_translation_suggestion_count(
+            self):
+        self._assert_community_contribution_stats_is_in_default_state()
+        translation_suggestion = (
+            self._create_translation_suggestion_with_language_code(
+                self.language_code))
+        # Assert that the translation suggestion count increased.
+        stats = suggestion_services.get_community_contribution_stats()
+        self.assertEqual(stats.question_reviewer_count, 0)
+        self.assertEqual(stats.question_suggestion_count, 0)
+        self.assertDictEqual(
+            stats.translation_suggestion_counts_by_lang_code,
+            {self.language_code: 1})
+        self.assertDictEqual(
+            stats.translation_reviewer_counts_by_lang_code, {})
+
+        suggestion_services.accept_suggestion(
+            translation_suggestion.suggestion_id, self.reviewer_id,
+            self.COMMIT_MESSAGE, 'review message')
+
+        stats = suggestion_services.get_community_contribution_stats()
+        self.assertEqual(stats.question_reviewer_count, 0)
+        self.assertEqual(stats.question_suggestion_count, 0)
+        self.assertDictEqual(
+            stats.translation_suggestion_counts_by_lang_code,
+            {self.language_code: 0}
+        )
+        self.assertDictEqual(
+            stats.translation_reviewer_counts_by_lang_code, {}
+        )
+
+    def test_reject_translation_suggestion_lowers_translation_suggestion_count(
+            self):
+        self._assert_community_contribution_stats_is_in_default_state()
+        translation_suggestion = (
+            self._create_translation_suggestion_with_language_code(
+                self.language_code))
+        # Assert that the translation suggestion count increased.
+        stats = suggestion_services.get_community_contribution_stats()
+        self.assertEqual(stats.question_reviewer_count, 0)
+        self.assertEqual(stats.question_suggestion_count, 0)
+        self.assertDictEqual(
+            stats.translation_suggestion_counts_by_lang_code,
+            {self.language_code: 1})
+        self.assertDictEqual(
+            stats.translation_reviewer_counts_by_lang_code, {})
```

Comment:
Super nit here & everywhere else, but it seems a bit inconsistent to order this code as:
- Question reviewer count
- Question suggestion count
- Translation suggestion count
- Translation reviewer count

Can we use a consistent ordering (maybe questions then translations, reviewers then suggestions)?

Reply:
Thanks for updating it. :)